lib/src/cognito_user.dart

```diff
@@ -246,12 +246,11 @@ class CognitoUser {
     final authParameters = {
       'REFRESH_TOKEN': refreshToken.getToken(),
     };
-    final keyPrefix = 'CognitoIdentityServiceProvider.${pool.getClientId()}';
     final lastUserKey = '$keyPrefix.LastAuthUser';
```

Comment:
You removed 
```dart
final keyPrefix = 'CognitoIdentityServiceProvider.${pool.getClientId()}';
```
And replaced it by 
```dart
  String get keyPrefix => 'CognitoIdentityServiceProvider.${pool.getClientId()}.$username';
```

Meaning that lastUserKey went from 
```dart
 'CognitoIdentityServiceProvider.${pool.getClientId()}.LastAuthUser';
```
To 
```dart
'CognitoIdentityServiceProvider.${pool.getClientId()}.$username.LastAuthUser';
```
Unfortunately, it's still referenced in [CognitoUserPool](https://github.com/furaiev/amazon-cognito-identity-dart-2/blob/4a0477dd236297100320154202b2fe8e255f2c19/lib/src/cognito_user_pool.dart#L81) as the former

Reply:
@malkovro That just a key name for the storage. I don't see any problems.
If you set your value using the key 'CognitoIdentityServiceProvider.PoolId1.Usernam1.**LastAuthUser**';
What's wrong to use the same key gettin your value back?

Reply:
How is the CognitoUserPool supposed to retreive the `LastAuthUser` username on line 81 with this key change ?